### DISCUSSION TITLE
Git LFS in Data Bricks Notebook

### DISCUSSION THREAD

**Discussion by qusai238** [git-lfs/git-lfs#5737] [Q&A]
Hello Everyone,

I am trying to upload a model file object with an extension of .xgb from datarbricks notebook to a branch on my repo. Although I specified in the .gitattributes files in my repo that I want the LFS to handle .xgb files and pointed it towards that extension I keep getting this error. Can anyone please help with this? Thanks. File size is 89MB which shouldn't be big for LFS to handle, I thought the limit was 100 MB, then files need to be handled using LFS.
![image](https://github.com/git-lfs/git-lfs/assets/74375388/dcd54e32-c526-4829-8004-f5c7a2ea09f2)

**Comment by bk2204** [comment]
Hey,

I don't think this is related to Git LFS at all.  It's hard to tell from this image, but I believe what you're seeing is an attempt to perform some sort of API request, in which case you're handling an object that's too large and the API is unhappy.  Moreover, I don't see any evidence in the Databricks documentation that it supports Git LFS at all (although it doesn't say either way), so it's probably ignoring your `.gitattributes` file and trying to upload the large file directly to the API.

All of this is a guess, because what you're showing us is an image and not plain text, so I can't search through it and see what's going on.  I'd recommend that you reach out to Databricks support and ask them what's going on, since it looks like the exception is coming from their side.
